components/views/src/main/res/layout/view_settings_left.xml

```diff
@@ -25,7 +17,7 @@
 
     <TextView
         android:id="@+id/settingsTitle"
-        style="@style/Body"
+        style="@style/Subhead1"
```

Comment:
Do we need to change this?  According to Figma specs we need to use style "Body"

Reply:
I know but there is no space and the text is too long in russia, so I had to use a smaller text size in ordee to fit.

Reply:
If text is too long, we ellipsize it.

Reply:
Now it should be okay. I ellipse the text and set the style to body.